### PR TITLE
Add Agile-Situation for better coding(Edited)

### PR DESCRIPTION
**Pull request by wodwine**
Add Agile-Situation for better coding and easy to work with people

### CODE REVIEW

File: agile-situation/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+In the situation below, which one of Agile is appropriate to solve the problem?
+
+1. I have 3 close friends, we group up the group in every work. One day, the teacher gave us the project. We discuss this and share the work for everyone in the group. But my one of my friends can’t do his/her work, another friend blame him/her. I think it’s not good enough for people to fight about this problem. So I think I have to use ………………………………….…… which is the one of Agile practices to solve this.
```
Comment (jbrucker) [inline agile-situation/README.md]: Lots of word use errors here.  Here is correction for just the first few sentences.
You need to correct the whole thing.

I have 3 close friends; we often form a group to do course work.  ["every work" - I doubt you do EVERY thing together]
One day, a teacher gave us a project.  We discussed it and divided the work among everyone in the group.

Comment (jbrucker) [inline agile-situation/README.md]: "have to use" - no one is forcing you to do anything, so "have to" is incorrect.

File: agile-situation/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+2. I have found the new things about python, it’s replace the old of my knowledge in python [Eg. format string in print(f”{a}”)], then I figure it out about one of the Agile practice, ……………………………….. It’s better for me though.
```
Comment (jbrucker) [inline agile-situation/README.md]: Several wording errors.  You frequently misuse "the".  Python is a proper noun (name of a language)
so it should have initial capital.  You must have learned this in English class.

"it's replace the old of my knowledge in python" = multiple problems.

File: agile-situation/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+4. Alex is a good programmer. He always writes a code with different way than other people, the code works perfectly, but other people can’t understand the code, so I suggest the Agile ……………………………. To him for improve!!
+5. One day in my project work, I do not know exactly the work I have to do because my friends already do my work and I don’t understand what they do, so I have to ……………………………….. Bt the Agile practice way.
+
+<details><summary>Answer</summary>
+
+Answer
+1. Tips1 : Blame doesn’t fix bugs.
+    Blaming other people can't solve the problem, it might cause the relation between people in the long term, the best way is show them how to do or what to do when the problem's happend instead of blaming.
+
+2. Tips7 : Learn the new; unlearn the old.
```
Comment (jbrucker) [inline agile-situation/README.md]: 2 formatting errors on this line.

File: agile-situation/README.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    The new things is importance for programmer because the world has gone too far for many many years since the computer was invented. The best is find the new things and replace the old things by the better things. Many of programmer do this all the time for improving themselve.
+3. Tips13 : Keep your project releasable at all times.
```
Comment (jbrucker) [inline agile-situation/README.md]: Same formatting errors here.

**Comment by wodwine** [comment]
I already resolved the README.md

**Pull request merged**